VaccinationSystem/VaccinationSystem/Data/Vaccine.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace VaccinationSystem.Data
+{
+    public class Vaccine
```

Comment:
Dodałem klucz i klucz obcy do Diseases, ale nie jestem pewien czy nie powinniśmy zrobić jeszcze inaczej. Brat mi tu mówi, że wiele szczepionek działa na kilka chorób, więc jakbyśmy się chcieli super trzymać faktów to powinniśmy zrobić wiele do wielu. Ale nie pamiętam jak, więc może później xD

Reply:
myślę, że lepiej nie trzymać się faktów spoza specyfikacji, bo to nam tylko życie utrudni;)